tensorflow/contrib/receptive_field/python/util/receptive_field.py

```diff
@@ -52,10 +52,22 @@ def _stride_size(node):
     stride_x: Stride size for horizontal direction (integer).
     stride_y: Stride size for vertical direction (integer).
   """
-  strides_attr = node.attr["strides"]
-  logging.vlog(4, "strides_attr = %s", strides_attr)
-  stride_y = strides_attr.list.i[1]
-  stride_x = strides_attr.list.i[2]
+  if node.op == "MaxPoolV2":
+    strides_input_name = node.input[2]
+    if not strides_input_name.endswith("/strides"):
```

Comment:
Is this safe? Could there possible be multiple strides tensors in the innermost scope, e.g. strides_1?

Reply:
@rmlarsen I think it is fine. The name is `node.input[2]` and the op is checked with `node.op == "MaxPoolV2"`.